spec/factories/review.rb

```diff
@@ -9,5 +9,11 @@
     factory :new_review_type do
       new_review_format true
     end
+
+    factory :twenty_four_month_review do
+      review_type '24-Month'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.